<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lsk_chaxun</class>
 <widget class="QWidget" name="lsk_chaxun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_db">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>卡号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_db_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>读卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>60</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查看日志</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>411</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>卡号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>711</width>
     <height>341</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>卡号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>性别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>状态</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
